scripts/update/Updater.php

```diff
@@ -20,10 +20,21 @@
 
 namespace oat\taoOffline\scripts\update;
 
+use oat\taoOffline\scripts\tools\byOrganisationId\RegisterTestCenterFormService;
+use oat\taoOffline\scripts\tools\byOrganisationId\RewriteTestCenterManagerService;
+use oat\taoOffline\scripts\tools\byOrganisationId\SetupTestCenterImporterByOrgId;
+
 class Updater extends \common_ext_ExtensionUpdater
 {
     public function update($initialVersion)
     {
         $this->skip('0.1.0', '2.2.1');
+
+        if ($this->isVersion('2.2.1')) {
+            $this->runExtensionScript(RewriteTestCenterManagerService::class);
```

Comment:
you need to write service registering here directly, without running of these scripts

Reply:
fixed